conformance-suites/1.0.2/conformance/extensions/oes-vertex-array-object.html

```diff
@@ -539,6 +539,8 @@
         testFailed("buffer removed too early");
       }
```

Comment:
Sorry for the delay replying. I don't think it's necessary to do the unbind at this point. colorBuffers[ii] and elementBuffers[ii] are only referenced by vaos[ii] and not any other entry in the vaos array. Therefore leaving vaos[ii] bound will not affect the deletion or unbinding of colorBuffers[ii+1] and elementBuffers[ii+1]. Unbinding the last bound vao at the end of the loop should be sufficient.